src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -49,4 +49,12 @@ public static void printRouteMessage(ArrayList<Bus> options) {
             System.out.println("none.\n\tThere are no direct bus routes. :(");
         }
     }
+
+    public static void printAllBusMessage(ArrayList<Bus> allBuses) {
+        System.out.println("\nThe buses available in NUS are: ");
+        for (Bus bus : allBuses) {
```

Comment:
Maybe you can print out only the 1st and the last stop of the buses here with a ... in the middle.

Reply:
might be a little weird because both AA1 and AA2 would show "PGP-> ...-> PGP" right now , but I will take note of that for future updates

Reply:
Oh yah! Okie!